src/widgets/history/HistoryComparison.php

```
@@ -98,87 +91,21 @@ private static function buildMetrics(RequestSummary $baseline, RequestSummary $t
-            );
-
-            $added = $difference->added;
-            $removed = $difference->removed;
-            $changed = $difference->changed;
-            $unchanged = $difference->unchanged;
-
-            if ($added + $removed + $changed === 0 && $baselineState !== $targetState) {
-                $changed = 1;
-            }
-
+        foreach (PanelComparison::between($baseline, $target, $panelLabels) as $panel) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Debug\Comparison\PanelComparison' in method 'buildPanels'. (undefined)

(StaticAccess)